src/Internal/EventStoreCatchUpSubscription.php

```diff
@@ -163,37 +162,19 @@ private function loadHistoricalEvents(): void
     private function subscribeToStream(): void
     {
         if (! $this->shouldStop) {
-            $eventAppeared = new class(Closure::fromCallable([$this, 'enqueuePushedEvent'])) implements EventAppearedOnSubscription {
-                private $callback;
-
-                public function __construct(callable $callback)
-                {
-                    $this->callback = $callback;
-                }
-
-                public function __invoke(
-                    EventStoreSubscription $subscription,
-                    ResolvedEvent $resolvedEvent
-                ): void {
-                    ($this->callback)($subscription, $resolvedEvent);
-                }
+            $eventAppeared = function (
+                EventStoreSubscription $subscription,
+                ResolvedEvent $resolvedEvent
+            ): void {
+                ($this->callback)($subscription, $resolvedEvent);
```

Comment:
What is `$this->callback` ? No such property exists.

Reply:
You tell me. It was there before. :) Not sure from where it came from and how to substitute it now.

Reply:
Check the argument that was previously passed to the anonymous class constructor. That's what you should use here.

Reply:
Guess I need to add the callback as first arg of the anonymous function? I'm not sure how the callback is now passed.

https://github.com/prooph/event-store-http-client/blob/master/src/Internal/EventStoreCatchUpSubscription.php#L166-L180